.devcontainer/contributor/devcontainer.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "Radius CLI - Contributor",
+    "image": "mcr.microsoft.com/devcontainers/base:ubuntu",
+    "features": {
+        "ghcr.io/devcontainers/features/go:1": {
+            "version": "1.21"
```

Comment:
Would it be OK for us to un-pin this? Go's tooling has been rock-solid for us across a variety of versions.

We already update `go.mod` when we need to. I'd rather not have a second thing to update.